CONTRIBUTING.md

```diff
@@ -1,45 +1,79 @@
-How the code in this repo works
-===
+#How the code in this repo works
+
 Hi! Glad you want to add code to this project. First, a brief overview of what's gone into this repo and some suggestions on how to get started using it.
 
-The frontend stack
----
+Formally, this repository contains a Redux app written with ES6 JavaScript, styled with Sass, and served with hotloading and module support from Webpack. It relies on a backend with a RESTful API, which is a Django server running PostgreSQL in Docker. 
+
+The section "Uh, where does the code even start?" gives a brief overview. The two parts after that detail the frontend and backend stacks, respectively.
```

Comment:
Can we change the name of this? Or am I the only one who is put off by the title of this section?